### PR TITLE
added developer story

### PR DESCRIPTION
**Pull request by ruthgrace**
Here's my crack at a developer perspective. Let me know if there are any changes you would like me to make. Thanks for maintaining this super helpful resource :)

### CODE REVIEW

File: sysadmin_101.rst [hunk 1]
```diff
@@ -187,6 +187,15 @@ These backends can be big, and have many considerations behind their design.
   his code will run on. He works closely with his Operations engineers to make sure his
   code is performant and on capacity planning.
 
+  .. _whats-developer-francesca:
+
+  * **Francesca is one of five students working on the website for her school's
+  internationally famous hackathon. Her and her friends run a Tornado web server
```
Comment (miketheman) [inline sysadmin_101.rst]: "Her and her friends" is probably better as "She and her friends..."

**Comment by miketheman** [comment]
Nice! One comment, and then I think we can merge.

File: sysadmin_101.rst [hunk 2]
```diff
@@ -187,6 +187,15 @@ These backends can be big, and have many considerations behind their design.
   his code will run on. He works closely with his Operations engineers to make sure his
   code is performant and on capacity planning.
 
+  .. _whats-developer-francesca:
+
+* **Francesca is one of five students working on the website for her school's
```
Comment (miketheman) [inline sysadmin_101.rst]: Build results are:
> 192: WARNING: Inline strong start-string without end-string.

The `**` at the start of the sentence is meant to bold something, and there's no associated ending `**` which is what is triggering this failure.

Comment (ruthgrace) [inline sysadmin_101.rst]: Ah, thank you! I've been confused about the error.

**Comment by miketheman** [comment]
Thanks!

**Pull request merged**